flask_multipass/providers/shibboleth.py

```diff
@@ -67,7 +67,14 @@ def _shibboleth_callback(self):
 
         if not attributes:
             raise AuthenticationFailed("No valid data received", provider=self)
-        return self.multipass.handle_auth_success(AuthInfo(self, **attributes))
+
+        # Define 'identifier' attribute if 'identifier_field' is defined.
+        # Required to use a LDAP identity provider with the SSO auth provider.
```

Comment:
I think it would make more sense to make the `identifier` key on the ldap side configurable.

Reply:
I basically agree... But looking at the modification involved, I found easier to do it in the Shibboleth provider... That said, I could look again because it took me some time to understand that it was necessary to change the UID field too and the `identifier` part may be trivial (it was my initial implementation in fact !)